docs/installation-contributors.md

```diff
@@ -8,11 +8,18 @@ title: Installation
 The following instructions are intended for contributors who want to try Metals
 and provide feedback. We do not provide support for day-to-day usage of Metals.
 
-# Global setup
```

Comment:
"Global setup" and "Per-project setup" were header 1 which makes them not appear in the sidebar

<img width="972" alt="screen shot 2018-06-23 at 11 39 22" src="https://user-images.githubusercontent.com/1408093/41808181-1a9e64ca-76da-11e8-8a22-9a3d2175992e.png">